CHANGLOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Support Wechat bot receiver(#168). @Gentleelephant
 - Support discord receiver(#169). @Gentleelephant
+- Update kube-rbac-proxy to 0.11.0(#186). @Gentleelephant
```

Comment:
Not a feature, it should be in the `Upgrades` section.

Reply:
![image](https://user-images.githubusercontent.com/18525465/210961711-903dfde7-be01-4db1-bd5a-a92b0d0a0939.png)